pkg/testgridanalysis/testreportconversion/testresult.go

```diff
@@ -112,19 +112,20 @@ func filterTestResults(
 		filteredResults = append(filteredResults, testResult)
 	}
 
-	return filterSuccessfulTestResults(filteredResults)
+	return filterSuccessfulTestResults(filteredResults, successThreshold)
```

Comment:
i'm still confused by why you're doing this, didn't it already get filtered on line 104 above?

Reply:
(alternatively, remove line 104?)